underscore.js

```diff
@@ -436,6 +436,18 @@
   _.toArray = function(obj) {
     if (!obj) return [];
     if (_.isArray(obj)) return slice.call(obj);
+    if (_.isString(obj)) {
+      // Keep surrogate pair characters together
+      // https://mathiasbynens.be/notes/javascript-unicode
+      return _.reduce(obj, function (output, char, idx) {
+        if (idx > 0 && obj[idx-1].match(/[\uD800-\uDBFF]/) && char.match(/[\uDC00-\uDFFF]/)) {
```

Comment:
Isn't the regex more concise? Also, isn't lookbehind + reduce more concise?

Apologies, but I'm new to contributing here, so am not sure yet what styles/techniques are preferred.